ssg/templates.py

```diff
@@ -27,23 +29,32 @@
     "puppet": templating_lang("puppet", ".pp", template_type.remediation, "puppet"),
     "sce-bash": templating_lang("sce-bash", ".sh", template_type.remediation, "sce")
 }
-preprocessing_file_name = "template.py"
 
-templates = dict()
+PREPROCESSING_FILE_NAME = "template.py"
+TEMPLATE_YAML_FILE_NAME = "template.yml"
 
 
-class Template():
+class Template:
     def __init__(self, template_root_directory, name):
+        self.langs = []
         self.template_root_directory = template_root_directory
         self.name = name
         self.template_path = os.path.join(self.template_root_directory, self.name)
```

Comment:
Wouldn't it be lovely to call this a `template_root`? While template path is not wrong, I think that root is better, as a template is a collection of files rather than one thing where you can point with a path and that's it.

Reply:
In these names there is usually a dichotomy of `file_name` and `path`. The name `template_root` does not immediately identify itself as a file name or a path and gives a false impression that there are sensible locations deeper inside this root path.